Cargo.lock

```diff
@@ -1702,21 +1690,37 @@ dependencies = [
 
 [[package]]
```

Comment:
We have two `swc_core` entries. I'll publish new versions of crates.

Reply:
Done. Please update crates so `swc_core` is not duplicated

Reply:
Thanks, updated PR to bump up deps.